sdk/core/azure-core/inc/azure/core/internal/log.hpp

```diff
@@ -5,7 +5,7 @@
 
 #include "azure/core/logging/logging.hpp"
 
-namespace Azure { namespace Core { namespace Logging { namespace Details {
-  bool ShouldWrite(LogClassification const& classification);
-  void Write(LogClassification const& classification, std::string const& message);
-}}}} // namespace Azure::Core::Logging::Details
+namespace Azure { namespace Core { namespace Logging { namespace Internal {
+  bool ShouldWrite(LogLevel level);
```

Comment:
Can we call these ShouldLog and Log? This makes it clear when looking at code when logging is happening.
Also, they take a LogLevel, not a WriteLevel.